src/ui/hooks/comments/useAddComment.tsx

```diff
@@ -25,7 +25,6 @@ export default function useAddComment() {
   }
 
   return (comment: Comment) => {
-    const temporaryId = new Date().toISOString();
```

Comment:
@jaril can you remember why we do this? I couldn't remember and removing it seemed fine 🤷

Reply:
I'm going to assume this is fine because we already removed it for replies in previous commit.